### DISCUSSION TITLE
Feature-Sliced Design in Nuxt

### DISCUSSION THREAD

**Discussion by sethonne** [nuxt/nuxt#30418] [Questions]
Hello! I've recently started working on a medium-sized project in Nuxt, but I'm very used to implementing a [Feature Sliced Design](https://feature-sliced.design/docs/guides/tech/with-nextjs)  type of file structure in my projects. I was wondering if something like this was also possible in Nuxt even with its built-in conventions such as the `/components` directory. 

I'd be happy to hear any insights and I'm definitely open to hearing other alternative conventional ways of file organization! Thanks!

**Comment by TheAlexLichter** [comment]
Try out [nuxt layers](https://nuxt.com/docs/getting-started/layers).

**Reply by sethonne** [reply]
Gotcha! Will try reading up on it. Thanks @TheAlexLichter
